app/models/event_schedule.rb

```diff
@@ -37,6 +39,16 @@ def replacement?
 
   private
 
+  def start_after_end_hour
+    return unless errors.blank? && event.program.conference.try(:end_hour)
```

Comment:
We do have validations to check presence of event, start_time and everything else. If event is `nil` `validates :event, presence: true` will fail, but `event.program.conference.try(:end_hour)` will still be executed and it will raise error and because of this our test were failing. So to avoid this `event.program.conference.try(:end_hour) `should be executed only when all our previous validations have passed therefore I added `unless errors.blank?`.